pkg/apis/nsx.vmware.com/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -41,12 +41,27 @@ type VPCNetworkConfigurationSpec struct {
 	// Must be Public or Private.
 	// +kubebuilder:validation:Enum=Public;Private
 	DefaultSubnetAccessMode string `json:"defaultSubnetAccessMode,omitempty"`
+	// ShortID specifies Identifier to use when displaying VPC context in logs.
+	// Less than equal to 8 characters.
+	// +kubebuilder:validation:MaxLength=8
+	// +optional
+	ShortID string `json:"shortID,omitempty"`
 }
 
 // VPCNetworkConfigurationStatus defines the observed state of VPCNetworkConfiguration
 type VPCNetworkConfigurationStatus struct {
 	// Conditions describes current state of VPCNetworkConfiguration.
 	Conditions []Condition `json:"conditions"`
+	// LBSubnets describes lb subnets info, now it is needed for AKO.
+	LBSubnets []LBSubnetInfo `json:"lbSubnets,omitempty"`
+}
+
+// LBSubnetInfo defines lb subnet info.
+type LBSubnetInfo struct {
```

Comment:
Maybe change it to VPCInfo in case we will support multiple VPCs and add more VPC status later.

Reply:
thanks, changed.